.gitignore

```diff
@@ -15,6 +15,7 @@ _site
 doc
 docker-env
 dist
+types
```

Comment:
<img width="680" alt="image" src="https://user-images.githubusercontent.com/13958569/223301036-85609b09-55dc-4f36-963f-7c3dd3c52cfb.png">
Maybe because I merged the master branch several weeks ago, and it's not the latest version in my local code base.